### PR TITLE
SUBMARINE-514. [WEB] Implement experiment information output page.

### PR DESCRIPTION
**Pull request by kobe860219**
### What is this PR for?
Implement experiment information output page.
Reference:https://www.comet.ml/pingsutw/testing/e60b90e9096c4c639502278620064992?experiment-tab=stdout

### What type of PR is it?
[ Improvement ]

### Todos
* [ ] - Task

### What is the Jira issue?
https://issues.apache.org/jira/browse/SUBMARINE-514

### How should this be tested?
https://travis-ci.org/github/kobe860219/submarine/builds/692466737

### Screenshots (if appropriate)
螢幕錄製 2020-05-29 下午3

### Questions:
* Does the licenses files need update? No
* Is there breaking changes for older versions? No
* Does this needs documentation? No

### CODE REVIEW

**Comment by kobe860219** [comment]
@liuxunorg @pingsutw @chiajoukuo @jasoonn Please help me review this pr. Thanks a lot!

File: submarine-workbench/workbench-web-ng/src/app/pages/workbench/experiment/experiment-info/outputs/outputs.component.ts [hunk 1]
```diff
@@ -17,15 +17,43 @@
...
+  isShowing = false;
 
-  ngOnInit() {}
+  constructor(private experimentService: ExperimentService, private nzMessageService: NzMessageService) {}
+
+  ngOnInit() {
+    this.getExoerimentLog();
+  }
+
+  getExoerimentLog() {
```
Comment (jasoonn) [inline submarine-workbench/workbench-web-ng/src/app/pages/workbench/experiment/experiment-info/outputs/outputs.component.ts:42]: A small typo XDD

File: submarine-workbench/workbench-web-ng/src/app/pages/workbench/experiment/experiment-info/outputs/outputs.component.ts [hunk 2]
```diff
@@ -17,15 +17,43 @@
...
+  podNameArr;
+  podLogArr;
+  logDetailArr;
+  isShowing = false;
 
-  ngOnInit() {}
+  constructor(private experimentService: ExperimentService, private nzMessageService: NzMessageService) {}
+
+  ngOnInit() {
+    this.getExoerimentLog();
```
Comment (jasoonn) [inline submarine-workbench/workbench-web-ng/src/app/pages/workbench/experiment/experiment-info/outputs/outputs.component.ts:39]: A small typo XDD

File: submarine-workbench/workbench-web-ng/src/app/pages/workbench/experiment/experiment-info/outputs/outputs.component.ts (hunk 1 above)
Comment (kobe860219) [reply submarine-workbench/workbench-web-ng/src/app/pages/workbench/experiment/experiment-info/outputs/outputs.component.ts:42]: I have renamed it. Thanks XD

File: submarine-workbench/workbench-web-ng/src/app/pages/workbench/experiment/experiment-info/outputs/outputs.component.ts (hunk 2 above)
Comment (kobe860219) [reply submarine-workbench/workbench-web-ng/src/app/pages/workbench/experiment/experiment-info/outputs/outputs.component.ts:39]: I have renamed it too.

**Comment by chiajoukuo** [comment]
Thank for your contribution. It looks good to me.

**Comment by liuxunorg** [comment]
Will merge if no more comments.

**Pull request closed without merging**